[Bridge/PhpUnit] Extract all the code but shebang from bin/simple-phpunit

JustBlackBird opened the pull request:
The change allow `simple-phpunit` to be included without `#!/usr/bin/env php` string outputted.

Also it's the first step to fix #27035 (the second one is to change phpunit bridge recipe).

The reason why I fix a closed issue is because PHPStorm don't want to parse output from simple-phpunit with extra shebang.

![phpstorm-phpunit](https://user-images.githubusercontent.com/1167086/57097433-6c9b2a00-6d20-11e9-9979-0c2c6d24147d.png)


| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #27035
| License       | MIT

JustBlackBird commented:
Not sure if I should fix code style... The code is still the same but moved to another file.

nicolas-grekas commented on src/Symfony/Bridge/PhpUnit/bin/simple-phpunit:
```diff
@@ -10,251 +10,4 @@
  */
 
-// Please update when phpunit needs to be reinstalled with fresh deps:
-// Cache-Id-Version: 2018-11-20 15:30 UTC
```
please keep the above in this file

JustBlackBird replied:
This actually is not needed because md5 hash will be recalculated on `src/Symfony/Bridge/PhpUnit/simple-phpunit.php `

nicolas-grekas replied:
right

nicolas-grekas commented on src/Symfony/Bridge/PhpUnit/bin/simple-phpunit:
```diff
@@ -10,251 +10,4 @@
-}
-
-exit($exit);
+require dirname(__DIR__).DIRECTORY_SEPARATOR.'simple-phpunit.php';
```
I'd suggest putting the new file in the `bin/` directory

JustBlackBird replied:
Done

nicolas-grekas reviewed (commented):
Thanks, no need to care about the CS I agree.
This should be submitted on master - it's not a real bug fix. We can merge it for 4.3 IMHO.
Pleas also open a PR for the recipe so we can discuss it at the same time.

JustBlackBird committed:
[Bridge/PhpUnit] Extract all the code but shebang from bin/simple-phpunit

The change allow `simple-phpunit` to be included without
`#!/usr/bin/env php` string outputted.

Also it's the first step to fix #27035 (the second one is to change
phpunit bridge recipe).

JustBlackBird commented:
@nicolas-grekas I've changed target to master branch and open a PR for flux recipes repo: https://github.com/symfony/recipes/pull/584

nicolas-grekas commented on src/Symfony/Bridge/PhpUnit/CHANGELOG.md:
```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * added `ClassExistsMock`
  * bumped PHP version from 5.3.3 to 5.5.9 
+ * split simple-phpunit bin into php file with code and a shel script
```
shell

JustBlackBird replied:
Fixed

fabpot commented:
Thank you @JustBlackBird.

JustBlackBird commented:
You're welcome @fabpot !
